docs/tutorials/fundamentals/tutorial-swapping-token.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Tutorial - Swapping UOS from Ultra to Ethereum'
+order: -99993
+outline: [0, 7]
+---
+
+# Tutorial - Swapping UOS from Ultra to Ethereum
+
+This tutorial guides you through the process of swapping UOS tokens from the Ultra network to UOS ERC-20 tokens on the Ethereum network using the Ultra Client. This process is designed for users who already have an Ethereum account and are familiar with cryptocurrency transactions.
```

Comment:
Aloso mention here that the tutorial is specifically for Mainnet, maybe even good to put it in the guide title